bugalink-frontend/pages/users/[id]/edit.tsx

```diff
@@ -85,9 +92,26 @@ export default function EditProfile({ data }) {
       setErrors(errors);
       if (Object.keys(errors).length === 0) {
         // Aquí puedes hacer la llamada a la API o enviar los datos a donde los necesites
-        console.log(
-          'Los datos del formulario son válidos. ¡Enviando formulario!'
-        );
+        const url = `users/${user.id}/edit`;
+
+        const formData = new FormData();
+        formData.append('photo', file);
+        formData.append('first_name', name);
+        formData.append('last_name', surname);
+
+        
+        axiosAuth
+          .put(url, formData)
+          .then((res) => {
+            signOut({
```

Comment:
But you are also logging out the user everytime they edit their profile (and potentially their photo). Then, you don't need to do the call to `useUser()` in the home page, that's an unnecessary call to the backend. Ideally, remove the `userPhoto` variable in the home page and keep it as it was

Reply:
Again reffer to my previous comment